styles/main-style.css

```diff
@@ -91,6 +92,10 @@ body{
     line-height: 1.5rem;
 }
 
+/* .sidebar>ul>li>a, .sidebar>h2, .logo, .nav-links{
+    color: var(--primary-color);
+} */
+
 .footer{
     display: flex;
     flex-direction: column;
```

Comment:
you could use a utility class for `display: flex; flex-direction:column;` since it is being used at most of the places